applications/areg-portal/backend/main.py

```diff
@@ -95,28 +91,90 @@ async def has_access(credentials: HTTPBasicCredentials = Depends(security)):
 # Operations
 
 
-@prefix_router.get('/live', response_model=str, tags=['test'])
-def live() -> str:
+@prefix_router.get('/antibodies', response_model=List[Antibody], tags=['antibody'])
+def get_antibodies(
+    page: Optional[int] = None, size: Optional[int] = None
+) -> List[Antibody]:
+    """
+    List Antibodies
+    """
+    return antibody_controller.get_antibodies(
+        page=page,
+        size=size,
+    )
+
+
+@prefix_router.post('/antibodies', response_model=None, tags=['antibody'])
+def create_antibody(body: AddUpdateAntibody) -> None:
     """
-    Test if application is healthy
+    Create an Antibody
     """
-    return test_controller.live()
+    return antibody_controller.create_antibody(
```

Comment:
I see here a useless indirection level. This is the controller IMO. @zsinnema thoughts?

Reply:
One of the positive points of this main layer is that when regenerating the api we don't lose the manual/custom changes that we might have on the controllers layer (an example of a custom change would be raising the appropriate HTTPException)

Reply:
@filippomc may be we can come up (in future) with an on the fly generated main.py :-)

Reply:
for now we have this extra layer for the purpose described by @afonsobspinto

Reply:
aren't the endpoints automatically generated by the fastapi generator?

Reply:
yes they are auto generated by the generator that's what you get in the `main.py`